src/clj/rems/application/events.clj

```diff
@@ -101,6 +101,12 @@
   (assoc EventBase
          :event/type (s/enum :application.event/rejected)
          :application/comment s/Str))
+(s/defschema ResourcesChangedEvent
+  (assoc EventBase
+         :event/type (s/enum :application.event/resources-changed)
+         (s/optional-key :application/comment) s/Str
```

Comment:
Base already has a comment. Accident, or making it optional (usually it's just empty)?

Reply:
Base doesn't have a comment.

Reply:
In the other events which have a comment, it's a mandatory field, but here it's optional. Any reason for that?

Reply:
The applicant does not need to add a comment when he/she is just creating an application but it would be nice if the handler can add a comment why he/she changed the resources.

Reply:
👌